src/victory-util/collection.js

```diff
@@ -22,5 +22,19 @@ export default {
 
   removeUndefined(arr) {
     return arr.filter((el) => el !== undefined);
+  },
+
+  getMaxValue(arr, value) {
+    const array = value ? arr.concat(value) : arr;
```

Comment:
Check out es6 rest params http://exploringjs.com/es6/ch_parameter-handling.html. You can make this method take an arbitrary number of additional args like
```
getMaxValue(arr, ...values) {
  arr = arr.concat(values);
  ...
}
```